docs/proposals/PROPOSAL_F.md

```diff
@@ -0,0 +1,112 @@
+
+### Artifact Manifest
+
+This artifact MAY have a manifest that references it with the media type that is specific to the compatibility interest group. For example, metadata for node feature discovery may take the following form:
+
+```json
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.oci.image.manifest.v1+json",
+  "artifactType": "application/vnd.oci.image-compatibility.v1",
+  "config": {
+    "mediaType": "application/vnd.oci.empty.v1+json",
+    "digest": "sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a",
+    "size": 2
+  },
+  "layers": [
+    {
+      "mediaType": "application/k8s.nfd.image-compatibility.spec.v1+json",
+      "digest": "sha256:4a47f8ae4c713906618413cb9795824d09eeadf948729e213a1ba11a1e31d052",
+      "size": 1710
+    }
+  ],
+  "annotations": {
+    "oci.opencontainers.image.created": "2024-01-02T03:04:05Z"
+  }
+}
+```
+
+### Artifact
+
+The artifact is a simple json structure of named groups and within them, key value pairs. The groups can provide a higher level organization, such as types of features or even versions. This organization is up to the compatibility interest group. Here is an example organizing NFD annotations within extractor groups (a random selection):
+
+```json
+{
+  "compatibilities": {
+    "cpu": {
+      "cpu-hardware_multithreading": true,
+      "cpu-pstate.status": "passive",
+      "cpu-pstate.turbo": false
+    },
+    "kernel": {
+      "kernel-selinux.enabled": true,
+      "kernel-version.major": 4           
+    },
+    "memory": {
+      "memory-numa": true,
+      "memory-nv.dax": true
+    },
+    "network": {
+      "network-sriov.capable": true,
+      "network-sriov.configured": true
+    }
+}
+```
+
+A similar spec for an HPC-oriented container might look like the following:
+
+```json
+{
+  "compatibilities": {
+    "cpu": {
+      "target": "arm64",
+    },
+    "mpi": {
+      "implementation": "openmpi",
+      "version": "4.1.6"           
+    },
+    "gpu": {
+      "enabled": true
+    },
+    "os": {
+      "vendor": "rocky",
+      "version": "8"
+    }
+}
+```
+
+As stated previously, the granularity of metadata provided is up to the compatibility interest group. The degree to which each metadata attribute is used is up to the plugin or tool within the context of a specific use case.
+
+## References
+
+- [Original planning document](https://docs.google.com/document/d/1nRUuW9i7NRdYXrUr8DQXLMETXwP1J9rw3TbpLTU9yho/edit)
+- [Use Cases](https://docs.google.com/document/d/1ULXHY0pdiLBlGZPN2Gj54XSvAo--UXKYr4opTrWSFMs/edit)
+- [Graph prototype added here](https://github.com/supercontainers/compspec/pull/6)
```

Comment:
list of fufilled use cases ( check box list of use cases)

Reply:
I removed that explicitly because we are largely throwing it away - most use cases in it are not being addressed. I'm not sure it was a great use of time.